src/API/src/config/config.controller.ts

```diff
@@ -33,4 +34,9 @@ export class ConfigController {
   async getMapStyles(): Promise<MapStyles> {
     return mapStyles;
   }
+
+  @Get('tile-server-overlays')
+  async getTileServerOverlays(): Promise<{ name: string; source: string }[]> {
+    return tileServerOverlays;
```

Comment:
From the AC's, it looks like these should be got from the Tileserver itself, not just from a static file.

Reply:
Here I am not so sure. I initially started working towards a solution in the manner that you describe. But Ray suggested that because the tile server and the API are separate containers, we might instead want to use a .json file to declare the overlays(mbtiles) we want to load into the tileserver. Then we use templates to build the config files required to push them onto the server - but that is my paraphrasing mixed with crude understanding @ray-millward-tessella

Reply:
Chris is correct - we want to decouple the two components, we may also choose to turn some layers on or off without removing them from the tile server. For the moment it's not trivial to get this list from the tileserver direct - we should just load it from a config file for now.